wordpress-mobile/WordPress-Android
# Add download package to jetpack.backup

Parent #13329 
The PR adds the `download` package to the `Jetpack.Backup` package to separate the download process from the daily backup views.

To test:
- Launch the app 
- Tap on Activity Log
- Tap on the more menu
- Tap the download backup option
- Ensure the backup download view is shown

PR submission checklist:
